sound/sound_library.h

```diff
@@ -72,11 +72,17 @@ class SoundQueueSingleton : public SoundQueue<16>, Looper, SaberBase {
       VoicePackVersionFile f;
       f.ReadInCurrentDir("voicepack.ini");
       found_version = f.voice_pack_version;
-      PVLOG_STATUS << "Sound library version " << found_version << " found.\n";
+      PVLOG_STATUS << "Voice pack version " << found_version << " found.\n";
     }
     if (found_version < required_version_) {
-      PVLOG_ERROR << "ERROR - Sound library version " << required_version_ << " required.\n";
-      ProffieOSErrors::error_in_font_directory(); // Make new error for voice pack?
+      if (SFX_mnum) {
+        PVLOG_ERROR << "** A version " << required_version_ << " voice pack is required to use the selected prop file.\n";
+        ProffieOSErrors::error_in_voice_pack_version();
+      } else {
+        PVLOG_ERROR << "** A voice pack is required to use selected prop file.\n";
+        PVLOG_ERROR << "** Add sounds to SD card and append 'common' to preset's font search path.\n";
```

Comment:
Two lines of errors seems like a lot. (and costs a few extra bytes of memory)
(Three if you count the one in ProffieOSErrors::voice_pack_not_fount.)
Maybe we should just add a URL to the POD where we can have a full explanation of the error and what to do about it?

Reply:
I don't ilke using TinyURL, since it makes it hard to know where you're going.
However, we could potentially set up redirects on the pod that have shorter paths.
Something like:
https://pod.hubbe.net/r/eifd  (r stands for redirects, eifd = error in font directory)

I'm not entirely sure if you can do redirect to anchors easily though. Might require javascript.
(Browsers tends to treat URLs and anchors as separate things.)

> Too bad serial monitor can't use hyperlinks :(

We'll have to make sure to add that for the master board. :)

Reply:
Did some testing, this seems to create a shortcut that works:
https://github.com/profezzorn/ProffieOSDocs/blob/main/r/eifd.md